### PR TITLE
Add parameter to disable sending first episode as next up

### PR DESCRIPTION
**Pull request by crobibero**
Fixes AndroidTV with 10.7, allows Vue to do what they want
https://github.com/jellyfin/jellyfin/pull/5026

### CODE REVIEW

**Review by nielsvanvelzen — APPROVED** [review]

**Review by barronpm — APPROVED** [review]

File: Emby.Server.Implementations/TV/TVSeriesManager.cs [hunk 1]
```diff
@@ -143,10 +143,33 @@ public IEnumerable<Episode> GetNextUpEpisodes(NextUpQuery request, User user, IE
             var allNextUp = seriesKeys
                 .Select(i => GetNextUp(i, currentUser, dtoOptions));
 
+            // allNextUp = allNextUp.OrderByDescending(i => i.Item1);
```
Comment (cvium) [inline Emby.Server.Implementations/TV/TVSeriesManager.cs:146]: Hmm?

Comment (crobibero) [reply Emby.Server.Implementations/TV/TVSeriesManager.cs:146]: Was a full revert from https://github.com/jellyfin/jellyfin/pull/4595
If desired I can remove the commented line.

Comment (cvium) [reply Emby.Server.Implementations/TV/TVSeriesManager.cs:146]: Yes please

**Comment by Uatschitchun** [comment]
Will there be a setting in jellyfin-web to change the query parameter?

**Comment by nielsvanvelzen** [comment]
It's up to the jellyfin-web contributors to implement this change, I guess changes to the javascript apiclient need to be made first.

**Pull request merged**